bioio/bio_image.py

```diff
@@ -281,41 +305,94 @@ def ends_with_ext(string: str) -> bool:
     @staticmethod
     def _get_reader(
         image: biob.types.ImageLike,
-        reader: biob.reader.Reader,
+        reader: Optional[
+            Union[Type[biob.reader.Reader], Sequence[Type[biob.reader.Reader]]]
```

Comment:
Maybe at this layer you could simplify by always passing in a sequence of Reader (or `None`)